src/components/SearchPanelControls.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Input from '@material-ui/core/Input';
+import InputLabel from '@material-ui/core/InputLabel';
+import FormControl from '@material-ui/core/FormControl';
+import SearchIcon from '@material-ui/icons/SearchSharp';
+import IconButton from '@material-ui/core/IconButton';
+import InputAdornment from '@material-ui/core/InputAdornment';
+
+/** */
+export class SearchPanelControls extends Component {
+  /** */
+  render() {
+    const { t, windowId } = this.props;
+    const id = `search-${windowId}`;
```

Comment:
It's possible to have multiple search controls per window; we should probably use the companion window's id instead.

Reply:
Ah great point. I've added a commit that does this